scripts/i18n/update-source.js

```diff
@@ -19,36 +23,42 @@ const dirsToCopy = [
 ]
```

Comment:
the last item is not a dir - maybe name is then wrong

I miss also the Sidebar yaml files ( the [docs](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/sidebars/doc-links.yaml), [tutorial](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/sidebars/tutorial-links.yaml), and [contributing](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/sidebars/contributing-links.yaml) )